pkg/render/guardian.go

```diff
@@ -260,6 +260,16 @@ func (c *GuardianComponent) volumes() []corev1.Volume {
 }
 
 func (c *GuardianComponent) container() []corev1.Container {
+	// UID 1001 is used in the guardian Dockerfile.
+	securityContext := securitycontext.NewBaseContext(1001, 0)
+	// minimize PS for deployment
```

Comment:
"minimize PZ for deployment" isn't a super descriptive comment, and won't be useful to folks reading this down the line.

I would expect something like this:

"Build a security context for the pod that will allow the pod to be deployed. Guardian is run in a namespace with a pod security standard of X, which requires that the security context meet certain criteria in order for pods to be accepted by the API server".